core/src/utils/qemu.c

```diff
@@ -33,10 +33,10 @@ int pg_util_cmdloop(const char *cmd, int timeout_s)
 	while (end.tv_sec - start.tv_sec < timeout_s) {
 		g_spawn_command_line_sync(cmd, NULL, NULL, &status, NULL);
 		if (g_spawn_check_exit_status(status, NULL))
-			return 0;
+			return -1;
```

Comment:
humm, so normally 0 mean that the function work, and here, if pg_util_cmdloop return 0 it mean that the function has fail. 
actually, it look like the old code was buggy because it was  returning 0 in case of success, which was wrong in the old context, but is the good behavior now.